execute_pipeline.py

```diff
@@ -137,5 +188,31 @@ def _var_replace(in_str, repl_vars):
   return new_str
 
 
+def _load_local_repo(private_repo_root, **pipeline_kwargs):
+  files_dict = {}
+  for root, dirs, files in os.walk(private_repo_root):
+      for fname in files:
+        path = os.path.join(root, fname)
+        rel_path = os.path.relpath(path, private_repo_root)
```

Comment:
That is a good point. I use _normalize_path method to turn path into one using slash as separator. I am not using filepath parts list because it is non-hashable and cannot be used a key in dict.

Currently, only relative paths of local files get normalized, and only makes the client side code platform-independent. More work needs to be done if we want to support using Windows for remote conductor.